go.mod

```diff
@@ -161,10 +161,10 @@ require (
 	gopkg.in/square/go-jose.v2 v2.3.1
 	gopkg.in/yaml.v2 v2.3.0
 	helm.sh/helm/v3 v3.2.4
-	istio.io/api v0.0.0-20200714164416-42acf9095d43
+	istio.io/api v0.0.0-20200721175012-ae75c7e9ae26
```

Comment:
I am pretty sure this will fail without olivers PR https://github.com/istio/istio/pull/25677

Reply:
I think you can remove api from this change

Reply:
This is reverted. Thanks